ansible_collections/arista/avd/tests/inventory/configlet_build_config/actual_output/actual_output.yml

```diff
@@ -0,0 +1,29 @@
+CVP_CONFIGLETS:
```

Comment:
is it on purpose that we store the `actual_output` in git?

Reply:
Yeah, sure. we can remove this actual output folder from Git.

Reply:
Removed the actual output files